extensions/graphite-emitter/src/main/java/io/druid/emitter/graphite/WhiteListBasedConverter.java

```diff
@@ -0,0 +1,288 @@
+package io.druid.emitter.graphite;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.google.common.base.Joiner;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.ImmutableSortedMap;
+import com.metamx.emitter.service.ServiceMetricEvent;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import java.util.SortedMap;
+import java.util.concurrent.TimeUnit;
+
+@JsonTypeName("whiteList")
+public class WhiteListBasedConverter implements DruidToGraphiteEventConverter
+{
+  @JsonProperty
+  private final SortedMap<String, Set<String>> whiteListDimsMapper;
+
+  @JsonProperty
+  private final boolean ignoreHostname;
+
+  @JsonProperty
+  private final boolean ignoreServiceName;
+
+  @JsonProperty
+  private final String namespacePrefix;
+
+  @JsonCreator
+  public WhiteListBasedConverter(
+      @JsonProperty("namespacePrefix") String namespacePrefix,
+      @JsonProperty("ignoreHostname") Boolean ignoreHostname,
+      @JsonProperty("ignoreServiceName") Boolean ignoreServiceName,
+      @JsonProperty("whiteListMap") SortedMap<String, Set<String>> whiteListDimsMapper
```

Comment:
should be take a file location instead, it will be very inconvenient to specify a big whitelist in the properties file in one line?